registries/_format/byte.md

```diff
@@ -12,7 +14,89 @@ layout: default
 
 Base type: `{{ page.base_type }}`.
 
-The `{{page.slug}}` format represents any sequence of octets encoded as a base64 string. This format entry is to ensure future versions of OpenAPI maintain compatibility with [OpenAPI 3.0.x](https://spec.openapis.org/oas/v3.0.0).
+The `{{page.slug}}` format represents any sequence of octets encoded as a base64 string as defined in [RFC4648](https://www.rfc-editor.org/rfc/rfc4648#section-4). This format entry is to ensure future versions of OpenAPI maintain compatibility with [OpenAPI 3.0.x](https://spec.openapis.org/oas/v3.0.0).
+
+### Upgrading from < 3.1 to >= 3.1
+
+#### Entire response body
+
+```yaml
+#OAS 3.0
+paths:
+  '/picture':
+    get:
+      responses:
+        '200':
+          content:
+            image/png:
+              schema:
+                type: string
+                format: byte
+
+#OAS 3.1
+paths:
+  '/picture':
+    get:
+      responses:
+        '200':
+          content:
+            image/png:
+              type: string
+              contentEncoding: base64
+```
```

Comment:
I think the OAS 3.0 example for "Entire response body" doesn't make sense. It first claims that the response will have `Content-Type: image/png`, and then claims that the response will be base64-encoded, that is plain text.

Reply:
right, what if we changed image/png instead to text/plain and added a contentMediaEncoding:  image/png ?

Reply:
@ralfhandl this is [what the spec says to do](https://spec.openapis.org/oas/v3.0.3#considerations-for-file-uploads), and [continues to say the analogous thing in OAS 3.1](https://spec.openapis.org/oas/v3.1.0#considerations-for-file-uploads) as noted with the additional paragraph I quoted elsewhere.

If there is a problem here, it is a problem in the spec, not with this PR.